docs/index.asciidoc

```diff
@@ -93,6 +93,21 @@ output {
 }
 -----
 
+**Example: Configuration for Winlogbeats to Output to Winlogbeat created Data Stream**
```

Comment:
This example is being inserted between the code block and explanation text of the previous example. Please move it to after the  "Customize data stream name" example.